actions/project.create.yaml

```diff
@@ -38,35 +46,30 @@ parameters:
     type: boolean
   parent_id:
     default: ""
-    description: Set the ID of the parent Domain/project. Leave as blank and it will use the "Default Domain"
+    description: Set the ID of the parent project.
     type: string
   admin_user_list:
-    required: false
+    required: true
     default: []
     type: array
     description: List of Users (IDs or Names) to give administrator access
-  stfc_user_list:
-    required: false
+  user_list:
+    required: true
     default: []
     type: array
-    description: List of Users (stfc domain) (IDs or Names) to give local user access
-  network_name:
-    required: true
+    description: List of Users (IDs or Names) to give local user access
+  networking_type:
+    description: Type of networking, internal or external
```

Comment:
Should be `internal`, `external` or `jasmin`
Where `jasmin` == external under the hood at the moment

Reply:
Yeah, this should be done in the backend code @meoflynn

Reply:
Would this be adding `jasmin`to the `RbacNetworkActions` enum to use?